### ISSUE TITLE
NavBar

### ISSUE THREAD

**Issue by emilystlaurent** [emilystlaurent/emilystlaurent.github.io#6]
I'm having issues trying to get my links for the nav to line up with my image for the nav. Currently, the links are below.

**Comment by saulbaizman-massart** [comment]
I don't understand. The nav links on the About page (about.html) look fine. See screenshot. Can you be more specific?
<img width="1531" alt="Screen Shot 2019-12-13 at 18 48 05" src="https://user-images.githubusercontent.com/57568420/70839095-33a85500-1dd9-11ea-8e62-9098946838f9.png">

**Comment by emilystlaurent** [comment]
its located in the new repository emilystlaurent.github.io

**Comment by saulbaizman-massart** [comment]
That's what I was talking about—[https://github.com/emilystlaurent/emilystlaurent.github.io/blob/master/about.html](https://github.com/emilystlaurent/emilystlaurent.github.io/blob/master/about.html). Can you provide a filename?

**Comment by emilystlaurent** [comment]
file:///Users/Emily%20St%20Laurent/Desktop/emilystlaurent.github.io/index.html

This is what it looks like in my browser. It's my index.html page in my
emilystlaurent.github.io folder on my desktop.

**Comment by saulbaizman-massart** [comment]
(Try to avoid referencing files on your computer; I can't see them. I can only see files in your repository.)

You're working on the header in index.html, and the issue is illustrated in the attached screenshot?

<img width="1531" alt="Screen Shot 2019-12-13 at 19 11 05" src="https://user-images.githubusercontent.com/57568420/70839771-643dbe00-1ddc-11ea-83b6-64b0aba9e116.png">

**Comment by emilystlaurent** [comment]
Correct! Sorry, there are still several things I'm unsure about using the
repository, so I'm not exactly sure how you are able to view them.

**Comment by saulbaizman-massart** [comment]
I see all files that are visible at [https://github.com/emilystlaurent/emilystlaurent.github.io](https://github.com/emilystlaurent/emilystlaurent.github.io).

As a collaborator, I can modify any files in your repository, too.

**Comment by saulbaizman-massart** [comment]
I took a look at your files. Please compare the markup and css in [about.html](https://github.com/emilystlaurent/emilystlaurent.github.io/blob/master/about.html) to [index.html](https://github.com/emilystlaurent/emilystlaurent.github.io/blob/master/index.html). Basically what I'm saying is, you appear to have already solved this problem, so what's the problem?

**Comment by emilystlaurent** [comment]
So this is a newer problem because the only way for me to get it correct in
the about.html was by using my grid incorrectly, which you helped me
troubleshoot last week. Because my grid was wrong on that template, I was
having major issues with my text (I had it at 1fr 4fr instead of 1fr 1fr
1fr 1fr). Currently, they both have different CSS pages if you recall from
Monday. I know you told me they should all have the same CSS sheet, which I
do intend on implementing but I wanted to work on the home page first.

I'm definitely coming to your office hours on Sunday, so we can also work
on troubleshooting this issue then if that's easier.

**Comment by saulbaizman-massart** [comment]
Emily, I recommend working on one page—about.html in this case—and then migrating the changes to index.html (and all of the other pages on your site). Don't fix individual problems on each page; it's a waste of time to reinvent the wheel. (And point all of your webpages to a single stylesheet.)

Does that make sense?

**Comment by emilystlaurent** [comment]
I have been working on one page since last class, the index.html. (I was
going to reformat the other pages around the updated index.html but I
wanted to get everything all set first.) I realized after starting it last
week that it'd be easier to work on one page at a time. I will
definitely switch to one style sheet and move away from index onto
about.html at this point in hopes that it will make things easier. I
appreciate the tips and I will see you on Sunday.

**Comment by saulbaizman-massart** [comment]
OK, see you Sunday! (By the way, I strongly suggest scheduling a time to meet Sunday rather than showing up whenever.)

**Comment by emilystlaurent** [comment]
Hi Saul,

I'm not sure if anyone already scheduled time with you, but I would love to
schedule some time at 2 pm to meet with you for office hours if
that's available. Please let me know! Otherwise, I can come earlier or
later.

Thanks,

Emily

**Comment by saulbaizman-massart** [comment]
2pm works. You're the first to make an appointment, so you may arrive earlier if you wish. I'll be on campus (somewhere on Tower 9) by 1pm.
